ml-agents/mlagents/trainers/settings.py

```diff
@@ -92,6 +92,11 @@ class ScheduleType(Enum):
     LINEAR = "linear"
 
 
+class ConditioningType(Enum):
```

Comment:
Like we discussed in the design doc, we *probably* don't need anything more than hyper, but it might still be worth keeping this just in case. For instances where the user finds out that the hypernetwork hurts performance on their task for whatever reason, it might be easier to disable it in the trainer than to rebuild their environment without goals.

Reply:
Made the default hyper and the no conditioning the second option

Reply:
I think it should be `hypernetwork` - is that too long?